production.Dockerfile

```diff
@@ -42,16 +43,8 @@ COPY . .
 # Precompile bootsnap code for faster boot times
 RUN bundle exec bootsnap precompile app/ lib/
 
-# Precompile assets
-# SECRET_KEY_BASE or RAILS_MASTER_KEY is required in production, but we don't
-# want real secrets in the image or image history. The real secret is passed in
-# at run time
-ARG SECRET_KEY_BASE=fakekeyforassets
-RUN ./bin/rails assets:precompile
-
-# TODO: This will work in Rails 7.1
 # Precompiling assets for production without requiring secret RAILS_MASTER_KEY
-# RUN SECRET_KEY_BASE_DUMMY=1 ./bin/rails assets:precompile
+RUN SECRET_KEY_BASE_DUMMY=1 ./bin/rails assets:precompile
```

Comment:
SECRET_KEY_BASE_DUMMY is introduced in Rails 7.1 for this purpose https://github.com/rails/rails/pull/46760